server/services/sqsService.js

```diff
@@ -102,9 +103,24 @@ module.exports = function createSqsService(offenderService, formService) {
               await offenderService.checkAndMergeOffenderNo(context, bookingId, transactionalDbClient)
             }
             break
-          case 'DATA_COMPLIANCE_DELETE-OFFENDER':
-            // TODO
+          case events.EVENT_DOMAIN_PRISONER_OFFENDER_SEARCH_PRISONER_RELEASED: {
+            logger.info(
+              { event },
+              `${events.EVENT_DOMAIN_PRISONER_OFFENDER_SEARCH_PRISONER_RELEASED}: Received payload`
+            )
+            const nomsNumber = event?.additionalInformation?.nomsNumber
+            if (!nomsNumber) {
+              logger.warn({ MessageId: message.MessageId }, 'MessageId was missing a nomsNumber')
+              break
+            }
+            const reason = event?.additionalInformation?.reason
+            if (!reason || reason !== 'RELEASED') {
+              logger.warn({ MessageId: message.MessageId, nomsNumber }, 'Reason was not "RELEASED"')
```

Comment:
Should this be a warning? Is it something which could happen often or a kind of error?

Reply:
good spot, thanks - no, lazy copy paste from the nomsNumber check above, fixed (set to info) and pushed